### PR TITLE
Add SPDX header

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Work in progress

### CODE REVIEW

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: firstrunwizard
+Upstream-Contact: Nextcloud <info@nextcloud.com>
+Source: https://github.com/nextcloud/firstrunwizard
+
+Files: .gitattributes .github/* .editorconfig babel.config.js .php-cs-fixer.dist.php package-lock.json package.json composer.json composer.lock */composer.json */composer.lock webpack.js stylelint.config.js .eslintrc.js .gitignore .jshintrc .l10nignore action/.gitignore action/package.json action/package-lock.json action/dist/index.js tests/* psalm.xml cypress/.env cypress/.eslintrc.js cypress/docker-compose.yml cypress/plugins/index.js cypress/tsconfig.json cypress.config.ts vendor-bin/*/composer.json stylelint.config.cjs vendor-bin/*/composer.lock .tx/config webpack.config.js tsconfig.json vite.config.ts vite.config.js js/vendor.LICENSE.txt krankerl.toml .npmignore cypress/fixtures/* postcss.config.js cypress/dockerNode.ts jest.config.js cypress.config.js relativeci.config.js .nextcloudignore .devcontainer/devcontainer.json .scrutinizer.yml .stylelintrc.js .tx/backport
```
Comment (susnux) [inline .reuse/dep5:6]: Is this some kind of template? I think it contains a lot of files that should rather have proper license tags, no?

File: .reuse/dep5 [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Files: js/*.js.map js/*.js js/*.mjs js/*.mjs.map js/templates/*.handlebars emptyTemplates/*
+Copyright: 2016 Nextcloud GmbH and Nextcloud contributors
+License: AGPL-3.0-or-later
+
+Files: css/firstrunwizard-main.css
+Copyright: 2024 Nextcloud GmbH and Nextcloud contributors
+License: AGPL-3.0-or-later
+
+Files: img/Nextcloud.mp4 img/Nextcloud.webm img/nextcloudHub.mp4 img/nextcloudHub.webm img/nextcloudLogo.svg
+Copyright: 2016 Nextcloud GmbH and Nextcloud contributors
```
Comment (susnux) [inline .reuse/dep5:27]:
```suggestion
Copyright: 2024 Nextcloud GmbH
```

File: .reuse/dep5 (hunk 1 above)
Comment (AndyScherzinger) [reply .reuse/dep5:6]: yes, I c&p from existing repos. So yes, some files for sure could have proper headers

File: .reuse/dep5 (hunk 2 above)
Comment (AndyScherzinger) [reply .reuse/dep5]: Fixed https://github.com/nextcloud/firstrunwizard/pull/1212/commits/d97ddac22725afc984bd01184b59fbe85ab5e6a1

File: .reuse/dep5 (hunk 1 above)
Comment (AndyScherzinger) [reply .reuse/dep5:6]: could be done in a follow up though

Comment (susnux) [reply .reuse/dep5]: Added / replaced old license tags on test files and config files to resolve this.
Also updated all workflows from templates to include the headers, see last two commits.

**Review by susnux — APPROVED** [review]

**Pull request merged**